src/Resources/config/doctrine_mongodb.xml

```diff
@@ -10,10 +10,12 @@
             <argument>%sonata.media.media.class%</argument>
             <argument type="service" id="doctrine_mongodb"/>
         </service>
+        <service id="Sonata\MediaBundle\Model\MediaManagerInterface" alias="sonata.media.manager.media" public="false"/>
```

Comment:
Public=false is not nesesery becouse it is default.

Reply:
Did you read this comment? https://github.com/sonata-project/SonataMediaBundle/pull/1781#discussion_r468680310.
The alias inherits the visibility from the service. Did you experienced a different behavior?

Reply:
I always use private service with public alias. Can you look at `Session`? Using `SessionInterface` with autowiring is even forced.